tests/tests/test_multi_tenancy.py

```diff
@@ -24,6 +24,7 @@
                           common_update_procedure
 from mendertesting import MenderTesting
 from tests import artifact_lock
+from boto3 import client
```

Comment:
Any chance you could import this inside the function where you use it? The thing is that this will fail for anyone who doesn't have this library, and testing on Amazon will rarely be done locally.

Reply:
yes, good point!